tests/unittest_nodes.py

```diff
@@ -1222,7 +1222,7 @@ def test_type_comments_with() -> None:
         """
     with a as b: # type: int
         pass
-    with a as b: # type: ignore
+    with a as b: # type: ignore[name-defined]
```

Comment:
Didn't saw that those these here are inside code blocks 😅
Doesn't matter now I guess.

Reply:
It's interesting that `mypy` picks up on those. Do you know if code in code blocks is checked? If not, this option also probably shouldn't?

Reply:
> It's interesting that `mypy` picks up on those. Do you know if code in code blocks is checked? If not, this option also probably shouldn't?

Mypy didn't notice those. I just searched for `type: ignore` and looked which ones didn't have error codes 😄

Reply:
If I'm not mistaken they did. I think I saw an error message about not having the error specified for these.

Reply:
Hmm 🤔 I just removed the comment and ran mypy. Didn't saw it.
Under the hood mypy uses ast parsing  and this would be just one string node. So hard to imagine that it would mark it.

Reply:
Probably my memory messing up 😅